lib/mime-type.js

```diff
@@ -68,6 +68,37 @@ module.exports = class MIMEType {
     return serialize(this);
   }
 
+  isJS() {
+    switch (this._type) {
+      default: return false;
```

Comment:
I'm surprised our ESLint doesn't catch this, but stylistically: default last; use curly braces around all blocks; indent all blocks.

Reply:
I think I've fixed this, unsure one what you consider a block for multiple case clauses.